cmake/std/atdm/apps/empire/EMPIRETrilinosEnables.cmake

```diff
@@ -12,6 +12,9 @@ ATDM_SET_CACHE(Gtest_SKIP_INSTALL TRUE CACHE BOOL)
 
 ATDM_SET_CACHE(Trilinos_ENABLE_Gtest FALSE CACHE BOOL)
 
+# Always set CUDA_LAUNCH_BLOCKING off for EMPIRE.
+unset(ENV{CUDA_LAUNCH_BLOCKING})
```

Comment:
I don't thinks this actually does what you want, does it?  This only unsets the env var `CUDA_LAUNCH_BLOCKING` when configuring Trilinos.  This will not impact running of EMPIRE tests.

Reply:
Ah, so changing this CMake fragment file won't make the change propagate to the `load_matching_env.sh` that gets installed?  That was the assumption I was working under.

> did you actually test this? 

No, I just created the PR to start the conversation.

> Can we create a new GitHub issue&hellip;?

Let's just keep the conversation here.  If this is going to be a substantial amount of work to make the change on the Trilinos side, we'll just make it on the BuildScripts side instead in the near term.

Reply:
@jmgate, I think an issue would be better to separate it from different implementation options in different PRs.  If there is any requirements analysis that needs to happen, then that should happen in an issue.